tests/webview/integration/test_archive_redirects.py

```diff
@@ -0,0 +1,15 @@
+import requests
+
+from tests import markers
+
+@markers.nondestructive
+@markers.parametrize("uuid", [("fea3130c-6e57-41b2-a00f-0267ffae273c")])
+def test_archive_redirect(archive_base_url, uuid):
```

Comment:
It's a bit unclear to me what this test is actually testing based on the name.

Would a better test name be `test_archive_redirect_contains_openstax_host_header`? 

We're completely fine with long test names. The idea is that they clearly convey the intent of the test.

Perhaps, the test itself could be made more generic and the header be included as a parameter? In this case, the test would become `test_archive_redirect_headers` or something like that.